### DISCUSSION TITLE
cannot import name 'dist' from 'gpytorch.kernels.kernel'

### DISCUSSION THREAD

**Discussion by HanyangHenry-Wang** [meta-pytorch/botorch#1605] [Q&A] [answered]
Hi, I use 'git clone' to obtain the most up-to-date version of Botorch in colab and it works quite well in the last few weeks. But today, when I imported botorch, there was an error: **cannot import name 'dist' from 'gpytorch.kernels.kernel'**. I am wondering what causes this issue. Thank you!

**Comment by Balandat** [comment, marked as answer]
This is b/c the most recent PRs (#1599) relies on a change that is not yet part of a recent gpytorch release yet. You'll need to install the latest gpytorch version from https://github.com/cornellius-gp/gpytorch as well if you want to use the botorch `main` branch. Alternatively you can install the latest botorch release version via `!pip install botorch`

**Reply by HanyangHenry-Wang** [reply]
Thank you for your kind reply. It works well :)
